bump-version.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
I'd move away from automating the version bumping. I think it's ok if we manually tag whenever we decide to release. Similar to https://github.com/kubeapps/kubeapps/blob/master/docs/developer/release-process.md#2---create-a-git-tag

Reply:
The good part of that is we remove a lot of code from this change and the repo in general.

The cost is that we could produce bad version tags, that are not semver or not monotonically increasing. Ideally, we should detect those to avoid releasing bad named versions or versions "from the past".

Reply:
Yeah, I guess we need to trust the operator that it's tagging a proper, valid version. We could also create the release a draft so it can be manually checked before releasing.

In any case, I kind of like how Kubeapps suggests to use some tooling to come up with a valid version, it's better than nothing. https://github.com/kubeapps/kubeapps/blob/master/docs/developer/release-process.md#2---create-a-git-tag

in following patches we should document our release process since now it has a manual component.